ejb3/src/main/java/org/jboss/as/ejb3/remote/EJBRemoteConnectorService.java

```diff
@@ -75,35 +70,14 @@ public void start(StartContext context) throws StartException {
         final AssociationService associationService = associationServiceInjectedValue.getValue();
         final Endpoint endpoint = endpointValue.getValue();
         Executor executor = executorService.getOptionalValue();
-        if(executor != null) {
+        if (executor != null) {
             associationService.setExecutor(executor);
         }
         RemoteEJBService remoteEJBService = RemoteEJBService.create(
             associationService.getAssociation(),
             remotingTransactionServiceInjectedValue.getValue()
         );
-        final ControlledProcessStateService processStateService = controlledProcessStateServiceInjectedValue.getValue();
```

Comment:
As of this PR, the only EJBRemoteConnectorService dependent is an EJB component with a Remote interface.  See EJBComponentDescription.setupRemoteView(...).  This may have made since when the service was started actively, but I don't see why it would be necessary now.